src/Components/Tabs/Tabs.js

```
@@ -0,0 +1,17 @@
+import React from "react";
+
+import {Tab} from "./Tab/Tab";
+
+import "./Tabs.scss";
+
+/**
+ * Renders the tabs.
+ * @return {JSX.Element}
+ */
+export function Tabs ({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)